client/src/legacy/LeftAndMain.Preview.js

```diff
@@ -453,8 +453,13 @@ $.entwine('ss.preview', function($){
       }
 
       // Mark url as a preview url so it can get special treatment
-       if (url) {
-          url += ((url.indexOf('?') === -1) ? '?' : '&') + 'CMSPreview=1';
+      if (url) {
+        let urlFrag = url.split('#');
+        let urlBits = urlFrag.shift().split(/[?&]/);
+        let urlBase = urlBits.shift();
```

Comment:
urlBits and urlBase should be using `const`
yes, we disabled eslint for legacy files but let's not let standards slip too much :D

Reply:
Neato, wasn't aware. Thanks, will fix :)

Reply:
no, it's fine actually :)

`const` means a constant reference point, you can still mutate/change it but cannot reassign it.
if you had done `urlBits = anotherBit` then it would complain.

Reply:
Right, so `const` isn't a `constant`. Javascript ✅ Gotcha.

Reply:
Yeah Javascript 👍 
To make a true constant, you'd need a recursive `Object.freeze()` like `deepFreeze` then go `const MY_CONSTANT = deepFreeze(anything);`

Reply:
Ironically (not actually ironically), webpack knew this already. Only thing to commit after change & build was the source file, dist unchanged! :)